handler/handler_test.go

```diff
@@ -30,11 +36,51 @@ func TestNew(t *testing.T) {
 		args      args
 		checkFunc func(http.Handler) error
 	}
+	rt := role.Token{
+		Domain:        "rt_domain",
+		Roles:         []string{"rt_role1", "rt_role2", "rt_role3"},
+		Principal:     "rt_principal",
+		TimeStamp:     time.Unix(1595908257, 0),
+		ExpiryTime:    time.Unix(1595908265, 0),
+		KeyID:         "",
+		Signature:     "",
+		UnsignedToken: "",
+	}
+	bc := access.BaseClaim{jwt.StandardClaims{
+		Audience:  "at_domain",
+		ExpiresAt: 1595908275,
+		IssuedAt:  1595908267,
+		Subject:   "at_principal",
+	}}
+	at := access.OAuth2AccessTokenClaim{
+		AuthTime:       0,
+		Version:        0,
+		ClientID:       "client_id",
+		UserID:         "",
+		ProxyPrincipal: "",
+		Scope:          []string{"at_role1", "at_role2", "at_role3"},
+		Confirm:        nil,
+		BaseClaim:      bc,
+	}
```

Comment:
Can we use mock here other than strucht?

Reply:
Fixed
https://github.com/yahoojapan/authorization-proxy/pull/66/commits/879805e0405252b7f0189ce6e109502390404c5b